perception/autoware_multi_object_tracker/lib/tracker/motion_model/ctrv_motion_model.cpp

```diff
@@ -203,15 +203,26 @@ bool CTRVMotionModel::limitStates()
   Eigen::MatrixXd P_t(DIM, DIM);
   ekf_.getX(X_t);
   ekf_.getP(P_t);
-  X_t(IDX::YAW) = autoware::universe_utils::normalizeRadian(X_t(IDX::YAW));
+
+  // maximum reverse velocity
+  if (X_t(IDX::VEL) < 0 && X_t(IDX::VEL) < motion_params_.max_reverse_vel) {
```

Comment:
Since max_reverse_vel is already negative, do we need to check `X_t(IDX::VEL) < 0` ?

Reply:
it is fixed to check the parameter is negative and the velocity is smaller than the parameter